.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
Maybe we do want to have blank issues enabled? cc @janpio

Reply:
How exactly does this affect things? Didn't know this existed.

Reply:
![image](https://user-images.githubusercontent.com/22195362/76864016-6bb2ae80-6886-11ea-976f-59be587459e1.png)

This would add the following line in the issue page

Reply:
Yes, give people an "out" and we can definitely handle the current (and expected) volume.

Reply:
Adjusted this in all PRs ✅